docs/concepts/what-is-a-codec-server.md

```diff
@@ -0,0 +1,37 @@
+---
+id: what-is-a-codec-server
+title: What is a Codec Server?
+sidebar_label: Codec Server
+description: Explanation and implementation of a remote encryption/decryption server.
+---
+
+A Codec Server is a feature that can perform additional levels of encoding and decoding on Payloads that are handled by [tctl](/docs/tctl/index) or the [Web UI](/docs/web-ui/how-to-use-a-list-filter-in-the-temporal-web-ui).
+
+The Web UI and tctl both use a default [Data Converter](/docs/concepts/what-is-a-data-converter), which is capable of serialization only.
+
+Codec Servers can encrypt, compress, and change the format of a Payload object.
+These measures can further secure your data.
+
+![](/img/tctl-diagram-codec-server.svg)
```

Comment:
This is a pretty good generic diagram...

I think there could be more labels that clear up the "Encryption" item and relationship with keys.

But it passes muster for the moment. Nice work.